pkgs/development/python-modules/zigpy-znp/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
```

Comment:
Just importing `lib` would do.

Reply:
I reverted the other change, is this fine to leave?

Reply:
It's just an unnecessary indirection. I'm not aware of any harm it does.